src/content/ui-api/management-api/credentials/index.md

```diff
@@ -60,19 +60,45 @@ The first of the two identifies the IAM role to be assumed by Giant Swarm staff
+When creating a workload cluster, there will be an [`AWSCluster`]({{< relref "/ui-api/management-api/crd/awsclusters.infrastructure.giantswarm.io.md" >}}) or [`AzureCluster`]({{< relref "/ui-api/management-api/crd/azureclusters.infrastructure.cluster.x-k8s.io.md" >}})resource created, containing the provider-specific configuration of your workload cluster.
 
-If you create your cluster manifest manually or via the [`kubectl gs template cluster`]({{< relref "/ui-api/kubectl-gs/template-cluster" >}}) command, you are free to adapt the `AWSCluster` to match your exact requirements before submitting the manifest to the API (e. g. via `kubectl apply`).
+If you create your cluster manifest manually or via the [`kubectl gs template cluster`]({{< relref "/ui-api/kubectl-gs/template-cluster" >}}) command, you are free to adapt the provider cluster resource to match your exact requirements before submitting the manifest to the API (e. g. via `kubectl apply`).
 
-To specify which credential secret to use, the custom resource definition (CRD) provides an attribute [`.spec.provider.credentialSecret`]({{< relref "/ui-api/management-api/crd/awsclusters.infrastructure.giantswarm.io.md#v1alpha3-.spec.provider.credentialSecret" >}}). The two sub-attributes `name` has to match the credential secret's resource name. The `namespace` sub-attribute accordingly must match the secret's namespace.
+In AWS, it is possible to reference your credential secret directly. The custom resource definition (CRD) provides an attribute [`.spec.provider.credentialSecret`]({{< relref "/ui-api/management-api/crd/awsclusters.infrastructure.giantswarm.io.md#v1alpha3-.spec.provider.credentialSecret" >}}). The two sub-attributes `name` has to match the credential secret's resource name. The `namespace` sub-attribute accordingly must match the secret's namespace.
 
 Note that this attribute has to be set _before_ submitting the manifest to the Management API. Otherwise, if the `AWSCluster` resource gets submitted without any `.spec.provider.credentialSecret` in place, our admission controllers will fill in default values. Read on to understand the defaulting logic.
 
+The `AzureCluster` field  [`.spec.identityRef`]({{< relref "/ui-api/management-api/crd/azureclusters.infrastructure.cluster.x-k8s.io.md#v1alpha4-.spec.identityRef" >}}) is set by the `azure-operator`. It references credential secrets indirectly by creating an `org-credential` based on the organization your workload cluster is created in.
+
 ## Defaulting
 
-If the cluster resource does not reference a provider credential secret to use (as explained above), our admission controller fills in some defaults, in this logical order
+If the cluster resource does not reference a provider credential secret to use (as explained above and as always the case on azure), our admission controller fills in some defaults, in this logical order
```

Comment:
> "If the cluster resource does not reference a provider credential secret to use (as explained above and as always the case on azure)"

This is hard to understand for be because of the negative. What is always the case on Azure now?

- the cluster resource does reference a provider credential secret to use
- the cluster resource does **not** reference a provider credential secret to use

Reply:
On azure, the defaulting will always take place because we can not explicitly set the credentials. I am not sure how to formulate this in a simple manner.

Reply:
I decided to remove this part, since it is already explained above and we state that it is explained above.